### DISCUSSION TITLE
Request traceability

### DISCUSSION THREAD

**Discussion by vinukumar-vs** [Sunbird-Ed/Community#366] [General Discussion]
All the issues from contributors/adopters are tagging against Sunbird-ED. So it will be verify difficult to give quick responses to the community if it continues like this.

So we are going to suggest the community to verify the logs against the request & identify where the issue exist. This is possible only if all the teams followed the request traceability & all the logs are carrying the requestID.

EPIC link: https://project-sunbird.atlassian.net/browse/SB-21560
Log format: https://project-sunbird.atlassian.net/wiki/spaces/DEB/pages/1883144193/Log+Format
Exception logs: https://project-sunbird.atlassian.net/wiki/spaces/DEB/pages/1882652679/Exception+logs

If all the teams followed this approach, we can recommend any adopters to check the logs to identify the issue & create the discussion threads in the respective Building Blocks rather than tagging to Sunbird-ED. 

This will helps the community to get quicker response for the queries by tagging to the right BB. 

So I would request all the BB's to validate & take necessary steps if not ready.

Please confirm the status whatever(Done, Needs Improvement, Not present).

@pallakartheekreddy @reshmi-nair r @anandp504 @SanthoshVasabhaktula @vrayulu  @rhwarrie @mmanimegalai @Krishnaj20

@swayangjit @rajeevsathish @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @madhucr @alok-os
@coolbung @parthlawate  @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata  @ashokreddy1208 @gandham-santhosh @kameshbhr

**Comment by rhwarrier** [comment]
@AmiableAnil let's pick this up for Lern 5.4 as necessary
